src/mono/mono/metadata/class-internals.h

```diff
@@ -1248,6 +1249,9 @@ mono_class_vtable_checked (MonoDomain *domain, MonoClass *klass, MonoError *erro
 void
 mono_class_is_assignable_from_checked (MonoClass *klass, MonoClass *oklass, gboolean *result, MonoError *error);
 
+void
+mono_class_signature_is_assignable_from_checked (MonoClass *klass, MonoClass *oklass, gboolean *result, MonoError *error);
+
```

Comment:
Is there a need for the _checked suffix if there is no version without the suffix ?